### DISCUSSION TITLE
How to add card like Swell admin dashboard

### DISCUSSION THREAD

**Discussion by tarikrital** [swellstores/community#522] [Help] [answered]
Welcome,

Using admin dashboard it is easy to add card by filling the fields below:

1. Card number
2. Expiration
3. CVC

In contrast, by using backend APIs the `POST` needs `token` which gives error below:

`{ errors: { token: { message: 'Required', code: 'REQUIRED' } } }`

How to bypass that for test mode, I mean just fake cards for testing purpose?

**Comment by logeshswell** [comment]
`token` is a required field to add a card and you can't bypass it with an incorrect value.
Instead, you can generate and pass token value as follows:
https://developers.swell.is/frontend-api/payments#create-a-card-token

**Reply by tarikrital** [reply]
That is `swell-js` but we use `swell-node`!

**Reply by logeshswell** [reply, marked as answer]
No, you could tokenize the card details using swell-js (client-side) only.

**Reply by tarikrital** [reply]
Thank you, @logeshswell. 🎯
